contrib/init/README.md

```diff
@@ -0,0 +1,5 @@
+Sample configuration files for:
```

Comment:
nit: it could be also useful to detail how to enable/start/stop.

Reply:
Something like this?

```bash
# Enable lnd to automatically start on system boot
systemctl enable lnd

# Start lnd
systemctl start lnd

# Restart lnd
systemctl restart lnd

# Stop lnd
systemctl stop lnd
```